bootstrap/kubeadm/controllers/kubeadmconfig_controller_test.go

```diff
@@ -890,7 +888,7 @@ func TestKubeadmConfigReconciler_Reconcile_DisocveryReconcileBehaviors(t *testin
 		name              string
 		cluster           *clusterv1.Cluster
 		config            *bootstrapv1.KubeadmConfig
-		validateDiscovery func(*bootstrapv1.KubeadmConfig) error
+		validateDiscovery func(g *WithT, kubeAdm *bootstrapv1.KubeadmConfig) error
```

Comment:
we can omit the argument names from the declaration (as it was before) `func(*WithT, *bootstrapv1.KubeadmConfig) error` these can be on the side of the definition.

Reply:
ok sgtm

Reply:
done